book/intro.md

```diff
@@ -307,11 +309,11 @@ enhancement][prog-enhance-def].
     likely increasing over time. In some markets like China, "super-apps" act
     like a mobile web browser for web-view-based games and widgets.
     
-So given this centrality, it's worth knowing how the web works. And in fact, the
-web is built on simple concepts: open, decentralized, and safe computing; a
-declarative document model for describing UIs; hyperlinks; and the User Agent
-model.[^useragent] It's the browser that makes these concepts real. The browser
-is the User Agent, but also the _mediator_ of the web's interactions and the
+So given this centrality, it's worth knowing how the web works. And in
+particular, how the principles I listed earlier are put into practice by the
+User Agent, i.e. *the web browser*.[^useragent] It's the browser that makes
+these concepts real. The browser is the User Agent, but also the _mediator_ of
+the web's interactions and the
```

Comment:
The change is good but I worry this is still too wordy. What about something like:

> And in particular, it's worth focusing on the browser, which is the user agent
> and the mediator of the web's interactions, which ultimately is what makes
> the web's principles real.